packages/belos/src/BelosGmresPolySolMgr.hpp

```diff
@@ -90,39 +91,53 @@ class GmresPolySolMgrPolynomialFailure : public BelosError {public:
 /// \ingroup belos_solver_framework
 /// \example BlockGmres/BlockGmresPolyEpetraExFile.cpp
 ///
-/// "Hybrid block GMRES" means that the solver first runs block GMRES.
-/// It stores the resulting coefficients, which form a matrix
-/// polynomial.  It then can reuse this polynomial for subsequent
-/// solves.  This avoids the cost of the inner products and norms in
-/// GMRES.  However, the resulting polynomial is not necessarily as
-/// effective as the equivalent number of GMRES iterations.
-///
-/// We call solvers that take this approach "seed solvers."  Belos
-/// implements both a Block GMRES seed solver (this class) and a
-/// CG-based seed solver (Belos::PCPGSolMgr).
+/// The GMRES polynomial solver manager can perform two types of linear
+/// solves. First the solver runs block GMRES, storing the resulting 
+/// coefficients (or roots), which can be used to form a matrix polynomial.
+/// It then can reuse this polynomial as either a surrogate operator, or
+/// as a preconditioner for an outer solver.
+/// By applying the GMRES polynomial as an operator or preconditioner, one 
+/// avoids the cost of the inner products and norms in GMRES, thus reducing
+/// communication costs.  
+//
+/// The GMRES polynomial can be created in conjunction with any standard preconditioner.
```

Comment:
Yes, all solvers take a preconditioner in the linear problem.  But since it is unusual to combine preconditioners, might some users assume that the polynomial can only be used _instead of_ their preconditioner rather than _in addition to_ their preconditioner?

Reply:
Fair point.  It's unusual for _Belos_ solvers to permit combining preconditioners.  It's more common for Ifpack2 solvers to take an "inner preconditioner."